src/main/java/com/xpandit/plugins/xrayjenkins/Utils/BuilderUtils.java

```diff
@@ -28,4 +31,12 @@ public static boolean isSupportedJobType(Class<? extends AbstractProject> jobTyp
                 || MavenModuleSet.class.isAssignableFrom(jobType);
     }
 
+    /**
+     * Utility method to check if the test execution key is invalid
+     * @param testExecKey the test execution key
+     * @return <code>true</code> if the test execution key matches the regex or is null/empty, <code>false</code> otherwise
+     */
+    public static boolean isTestExecKeyInvalid(String testExecKey) {
+        return (StringUtils.isNotEmpty(testExecKey) || testExecKey.trim().matches(BETWEEN_BRACES_REGEX));
```

Comment:
Is this inverted? Should it be isEmpty()?
if testExecKey key it's null, _isNotEmpty_ will return false and then you will get an NPE